src/main/resources/db/migration/V006__Change_role_field_length.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE user MODIFY COLUMN ROLE varchar (50) not null;
```

Comment:
It's no problem to create a single migration file and add all changes in it (no need every change to be in a separate migration)